### DISCUSSION TITLE
[DDD START!] Chapter 1. 도메인 모델 시작

### DISCUSSION THREAD

**Discussion by jojaeng2** [BDD-CLUB/challenge-every-week-posting#548] [4. 기타 글 모음]
DDD를 설명해주는 책을 읽기 시작했습니다 
왜이렇게 공부할게 많을까요? 
https://blog.naver.com/ds4ouj/223473284791

**Comment by gusah009** [comment]
저는 조쟁 블로그로 DDD 공부하겠습니다 😊

**Comment by shkisme** [comment]
오 저도 최범균님의 DDD Start 책을 읽어볼까 고민했었는데 참고하도록 하겠습니다 👍
